test/test_utils.jl

```diff
@@ -189,3 +189,67 @@ end
     # `CSV` produces `SentinelArrays` data
     @test scatter(ChainedVector([[1, 2], [3, 4]]), 1:4) isa Plot
 end
+
+@testset "Best legend position" begin
+    x = 0:0.01:2
+    plt = plot(x, x, label = "linear")
+    plt = plot!(x, x .^ 2, label = "quadratic")
+    plt = plot!(x, x .^ 3, label = "cubic")
+    @test Plots._guess_best_legend_position(:best, plt) == :topleft
```

Comment:
```suggestion
    @test Plots._guess_best_legend_position(:best, plt) === :topleft
```

And :arrow_down: as well.